### PR TITLE
feat: put liquid templates in ts

### PR DESCRIPTION
**Pull request by danielgent**
Now there's no async upload step this _could_ be done at some build time but I don't think it's worth it :smile:

### CODE REVIEW

File: src/templates/email/ts/code-v2.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const codev2 = `
+export const codeV2 = `
```
Comment (danielgent) [inline src/templates/email/ts/code-v2.ts:1]: I quite like this to show that the script worked :smile: 

The other two template's typescript files are the same as the ones I manually built :sunglasses:

File: textfile2ts.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// duplicate of build-tailwind: use this there!
```
Comment (danielgent) [inline textfile2ts.js:1]: :eyes: something I could do next. this seems to work

File: src/controllers/email.ts [hunk 3]
```diff
@@ -7,8 +7,8 @@ import sv from "../locales/sv/default.json";
 import nb from "../locales/nb/default.json";
 import it from "../locales/it/default.json";
 import {
-  codev2,
-  linkv2,
+  codeV2,
+  linkV2,
```
Comment (danielgent) [inline src/controllers/email.ts:11]: I've gone against my own word and increased the diff here *BUT* I quite like this to show that the script worked :smile: 

The other two template's typescript files are the same as the ones I manually built :sunglasses:

Comment (danielgent) [reply src/controllers/email.ts:11]: Ideally soon I'll snapshot all these emails so we'd know that this PR didn't break anything :sunglasses:

**Review by markusahlstrand — APPROVED** [review]

**Pull request merged**